### PR TITLE
Swift Package Manager

### PR DESCRIPTION
**Pull request by maxxfrazer**
- [x] Added Swift Package Manager
- [x] Fixed Travis CI issues

### CODE REVIEW

**Comment by maxxfrazer** [comment]
@RamotionDev any chance to get this PR merged? Your latest commits implemented some of this, so I'm assuming you didn't see it yet?

**Comment by maxxfrazer** [comment]
@igork-ramotion Is there someone who can look at this PR?

File: Package.swift [hunk 1]
```diff
@@ -29,7 +29,8 @@ import PackageDescription
 let package = Package(
     name: "PaperOnboarding",
     platforms: [
-        .iOS(.v10)
```
Comment (igork-ramotion) [inline Package.swift:32]: Hey!) Good work, but let's keep .v10. Other things look good 
Thank You 👍🏻

Comment (maxxfrazer) [reply Package.swift:32]: can change, podspec says iOS version is 8 though, want to have swiftPM version different?

Comment (igork-ramotion) [reply Package.swift:32]: yeah let's change, we updated version in Readme. And will update it in podspec later 
Hope nobody already not targeting to iOS 8 :)

Comment (maxxfrazer) [reply Package.swift:32]: Makes sense, I'd be surprised if anyone was!
Just updated it now 👍

**Pull request merged**